vignettes/social_smell_showcase.Rmd

```diff
@@ -94,7 +97,16 @@ As stated in the introduction, we need both git log and at least one communicati
 
 ## Parse Gitlog
 
+Recommended perceval version: 0.12.24
 To get started, we use the `parse_gitlog` function to extract a table from the git log. You can inspect the `project_git` variable to inspect what information is available from the git log. 
+Reminder: the parse_gitlog function has a path.expand function, Expand paths (e.g. "~/Desktop" => "/Users/someuser/Desktop")
```

Comment:
This is good docs, but this should be added directly to the function parse_gitlog instead of the notebook, above the function definition on R/parse.R. Could you move this there instead?

Reply:
This is already in there, I added to the showcase as reminder because I think people might not know that parser_gitlog function will expand the path when they running the showcase.